package.json

```diff
@@ -27,11 +27,11 @@
     "@types/jest": "^23.1.1",
     "@types/uuid": "^3.4.3",
     "angular2-virtual-scroll": "^0.4.5",
-    "core-js": "^2.5.3",
-    "dropzone": "^5.4.0",
     "chai": "^4.1.2",
     "chance": "^1.0.16",
-    "cypress": "^3.0.2",
+    "core-js": "^2.5.3",
+    "cypress": "^3.1.0",
```

Comment:
Didn't notice this until now, but it seems that we've had cypress listed as a regular dependency. Can you move it over to devDependencies so we can get that taken care of in the process of this PR?